browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -5663,6 +5728,91 @@ index 32d32e59b24265286710a5f3f7053e1816ad9e3b..34f7e01b612710a817debb7f8d25d6bd
  int Frame::preferredHeight() const
  {
      Document* document = this->document();
+diff --git a/Source/WebCore/page/wpe/DragControllerWPE.cpp b/Source/WebCore/page/wpe/DragControllerWPE.cpp
+new file mode 100644
+index 0000000000000000000000000000000000000000..e86e4295101a0a3e9be64681a84eb16b8bfe49d8
+--- /dev/null
++++ b/Source/WebCore/page/wpe/DragControllerWPE.cpp
+@@ -0,0 +1,79 @@
++/*
++ * Copyright (C) 2007-20 Apple Inc.  All rights reserved.
```

Comment:
It might be easier to rename DragControllerGtk.cpp  to DragControllerGlib.cpp  and use it in both linux builds. This was they wouldn't diverge at least. But up to you.

Reply:
I wasn't able to get it to easily compile without a much larger diff. I am not super worried about this code diverging, because we would probably rather match the mac behavior than whatever gtk decides to do. I'll try a follow up patch to see how it looks though.